Gamenights/NewGameResultViewController.swift

```diff
@@ -15,6 +15,11 @@ class NewGameResultViewController: UIViewController {
     @IBOutlet weak var notesTextView: UITextView!
 
     var groupId: String?
```

Comment:
I didn't want to change too much code and contaminate this PR. :)
refactor next.  but yeah, should I merge #40 first?  that one needs to go in first